docs/datamodel/links.rst

```diff
@@ -340,10 +341,10 @@ A user can like multiple movies. And in the absence of an ``exclusive``
 constraint, each movie can be liked by multiple users. Thus this is a
 *many-to-many* relationship.
 
-Filtering, limiting, and ordering links
+Filtering, ordering, and limiting links
```

Comment:
I thought it might be better to list "filtering, ordering, and limiting" in a consistent order that reflects the order in which they are used in the select syntax.